apps/lend/src/store/createAppSlice.ts

```diff
@@ -107,26 +109,18 @@ const createAppSlice = (set: SetState<State>, get: GetState<State>): AppSlice =>
     state.updateGlobalStoreByKey('api', api)
     state.updateGlobalStoreByKey('isLoadingCurve', false)
 
-    const { owmDatasMapper } = await state.markets.fetchMarkets(api)
+    await state.markets.fetchMarkets(api)
```

Comment:
It may be worth querying this data in parallel, without waiting for the first query to be executed
```suggestion
    await Promise.all([
        state.markets.fetchMarkets(api)
```

Reply:
No, I need to call the market list first, which I use to get a list of tokens to call usdRates on.